shared/src/assets/icons/solAssets.tsx

```diff
@@ -4,31 +4,41 @@ type SVGProps = React.SVGAttributes<SVGElement>;
 
 export const SolanaLogo: React.FC<SVGProps> = (props) => (
   <svg
-    width="41"
-    height="32"
-    viewBox="0 0 41 32"
+    width="56"
+    height="56"
+    viewBox="0 0 56 56"
     fill="none"
     xmlns="http://www.w3.org/2000/svg"
     style={props.style || { padding: 4 }}
     {...props}
   >
-    <g clipPath="url(#clip0_2204_947)">
+    <path
+      d="M0 28C0 12.536 12.536 0 28 0C43.464 0 56 12.536 56 28C56 43.464 43.464 56 28 56C12.536 56 0 43.464 0 28Z"
+      fill="#14F195"
+      fill-opacity="0.12"
```

Comment:
need to change properties like `fill-opacity` to `fillOpacity` and `clip-path` to `clipPath`